index.html

```diff
@@ -178,21 +178,39 @@ <h2>About Me</h2>
               I'm always interested in hearing about new projects, so if you'd
               like to chat please get in touch.
             </p>
-            <form action="" id="form-contact">
-              <input class="form-text" type="text" placeholder="Full name" />
+            <form
+              id="form-contact"
+              method="POST"
+              action="https://formspree.io/f/xlekdbpd"
+            >
+              <input
+                class="form-text"
+                type="text"
+                name="name"
+                placeholder="Full name"
+                maxlength="50"
+                required
+              />
+
               <input
                 class="form-text"
                 type="email"
+                name="email"
                 placeholder="Email address"
+                required
               />
+
               <textarea
                 class="form-textarea"
-                name=""
                 id=""
```

Comment:
- [ ] Please remove the emty attributes, it is not in the HTML CSS best practices to commit an empty code to the codebase, Please remove this id attribute if you do not need it